pkg/operator/ceph/object/rgw-startup-probe.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
What's the difference between the startup and readiness probe? Could we refactor so there is a common script, with parameters for the differences?

Reply:
The difference is in how return 500 is handled. I suppose 1 script is fine with nothing more than that change.